lib/imgix/path.rb

```diff
@@ -61,33 +37,12 @@ def defaults
       self
     end
 
-    def method_missing(method, *args)
-      key = method.to_s.gsub("=", "")
-      if args.empty?
-        return @options[key]
-      elsif args.first.nil? && @options.key?(key)
-        @options.delete(key) and return self
-      end
-
-      @options[key] = args.join(",")
-      self
+    def ixlib(lib_version)
+      @options[:ixlib] = lib_version
     end
 
-    ALIASES.each do |from, to|
-      define_method from do |*args|
-        warn "Warning: `Path.#{from}' has been deprecated and " \
-             "will be removed in the next major version (along " \
-             "with all parameter `ALIASES`).\n"
-        send(to, *args)
-      end
-
-      define_method "#{from}=" do |*args|
-        warn "Warning: `Path.#{from}=' has been deprecated and " \
-             "will be removed in the next major version (along " \
-             "with all parameter `ALIASES`).\n"
-        send("#{to}=", *args)
-        return self
-      end
+    def ixlib=(lib_version)
+      @options[:ixlib] = lib_version
```

Comment:
What's the difference between this and the method on L40?

Reply:
They differ in how they are called; defining both gives us flexibility (if we want it). Here is a short example:

Small note, `p` in ruby "puts" the inspect string of an object, _ruby, amirite?_ ha
```ruby
class House
  def initialize
  end

  def address(addr)
    @address = addr
  end

  def address=(addr)
    @address = addr
  end
end

h = House.new
h.address = "423 Tehama"
p h # <House:0x00007faa328fef88 @address="423 Tehama">

h.address("5678 Market")
p h # <House:0x00007f96c4047380 @address="5678 Market">
```

Reply:
Ah, missed that part. In that case I'm good with this 👍